packages/zowe-explorer-api/__mocks__/mockCreators/shared.ts

```diff
@@ -92,3 +93,50 @@ export function createTeamConfigMock(): imperative.IConfig {
         autoStore: true,
     };
 }
+export class ExtenderAttr implements IAttributesProvider {
+    public fetchAttributes(): AttributeInfo {
+        const keys = new Map<string, { displayName?: string; description?: string; value: any }>([
+            [
+                "DescriptionAndDisplayName",
+                {
+                    displayName: "DisplayName",
+                    description: "Description",
+                    value: "Value1",
+                },
+            ],
+            [
+                "NoDescNoDisplayName",
+                {
+                    value: "Value2",
+                },
+            ],
+            [
+                "descNoDN",
+                {
+                    description: "Description - No Display Name",
+                    value: "Value3",
+                },
+            ],
+            [
+                "DNnoDesc",
+                {
+                    displayName: "DisplayName - No Description",
+                    value: "Value4",
+                },
+            ],
+        ]);
+
+        return [
+            {
+                title: "Dummy Extender",
+                reference: "https://github.com/zowe/zowe-explorer-vscode",
+                keys,
+            },
+        ];
+    }
```

Comment:
Typically I think we keep the return values of mocks in the `__mocks__` folder as empty/minimal as possible. If the return values are mocked within individual unit tests, it makes them easier to read.

Would it make sense for this function to return an empty array here, and perhaps the array can be moved to `DatasetAttributeProvider.unit.test.ts`?